frontend_tests/node_tests/user_groups.js

```diff
@@ -56,13 +56,14 @@ run_test("user_groups", () => {
     user_groups.update(update_des_event);
     assert.equal(user_groups.get_user_group_from_id(admins.id).description, "administer");
 
-    blueslip.expect("error", "Unknown group_id in get_user_group_from_id: " + all.id);
-    assert.equal(user_groups.get_user_group_from_id(all.id), undefined);
-
+    assert.throws(() => {
+        user_groups.get_user_group_from_id(all.id);
+    }, "Unknown group_id in get_user_group_from_id:");
```

Comment:
This is incorrect usage of [`assert.throws`](https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message).

> `error` cannot be a string. If a string is provided as the second argument, then `error` is assumed to be omitted and the string will be used for `message` instead. This can lead to easy-to-miss mistakes.

Reply:
Fixed.